Loop/Window Management/WindowDirection.swift

```diff
@@ -23,7 +23,7 @@ enum WindowDirection: String, CaseIterable, Identifiable, Codable {
     case initialFrame = "InitialFrame"
     case hide = "Hide"
     case minimize = "Minimize"
-    case macOSCenter = "MacOSCenter"
+    case macOSCenter = "macOSCenter"
```

Comment:
Use "MacOSCenter" instead of "macOSCenter" here. If renamed, this will break compatibility with exporting/importing keybinds from previous versions of Loop.

Reply:
Restored.